vendor/example/windsensor.yaml

```diff
@@ -141,11 +141,15 @@ firmwareProgramming:
 # Product and data sheet URLs (optional)
 productURL: https://example.org/wind-sensor
 dataSheetURL: https://example.org/wind-sensor/datasheet.pdf
+# URL for onboarding device (optional).
+# Can be a link to your own documentation or from us. Do not advertise the device in any way.
```

Comment:
> Can be a link to your own documentation or from us. Do not advertise the device in any way.

Improved text: `Link to simple, easy onboarding instructions for the device. Please do not use this for marketing or overly technical documents like a data sheet.`